plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ConfigureDependentFrameworksListener.java

```diff
@@ -109,12 +109,10 @@ private void configureDocker(IServer server, ADBInfo adb) {
 						.name(getNextName(server, mgr)).tcpHost(dockerHost);
 				String tlsVerifyString = adb.env.get("DOCKER_TLS_VERIFY");
 				boolean tlsVerify = (Integer.parseInt(tlsVerifyString) != 0);
```

Comment:
afaik just the presence of DOCKER_TLS_VERIFY is enough - the value does not matter.